ui/src/dialogs/AboutDialog.js

```diff
@@ -51,7 +51,7 @@ const AboutDialog = ({ open, onClose }) => {
                   <TableCell align="left">
                     <Link
                       href={`https://github.com/navidrome/navidrome/releases/tag/v${
```

Comment:
This is not what I meant. In the case of SNAPSHOTS, we want the link to be a list of commits since the release, something like `https://github.com/navidrome/navidrome/compare/v0.40.0...5f7c7393`, not just point to the release tag page.